plugins/quota/rest.go

```diff
@@ -114,9 +169,81 @@ func quotaEnableHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func quotaDisableHandler(w http.ResponseWriter, r *http.Request) {
-	// implement the help logic and send response back as below
+	// Collect inputs from URL
+	p := mux.Vars(r)
+	volName := p["volname"]
+
 	ctx := r.Context()
-	restutils.SendHTTPResponse(ctx, w, http.StatusOK, "todo: quota disable")
+	logger := gdctx.GetReqLogger(ctx)
+
```

Comment:
Lock must be held here, instead of in the txn step. Any call to the store such as below one must be under lock. See `glusterd2/commands/volumes/volume-start.go` for example.

Reply:
done.

Reply:
I don't see it yet.

Reply:
I did force push it. I'll check it for else were. In the mean time I'll push it again.

Reply:
Done. have pushed it.